makeup/html/private_office.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>CashTan</title>
+    <link rel="stylesheet" href="private_office.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.10/js/all.js"
+            integrity="sha384-slN8GvtUJGnv6ca26v8EzVaR9DC58QEwsIk9q1QXdCU8Yu8ck/tL/5szYlBbqmS+"
+            crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
+    <link rel="shortcut icon" href="images/cashtan_favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width">
+
+</head>
+
+
+<body>
+
+<div class="page_wrapper">
+    <header>
+        <!--меню-->
+        <div class="up-menu">
+            <div class="main-menu">
+                <div class="leaf-name">
+                    <a href="#" class="list-name">
+                        <img class="logo-menu" src="images/cashtan-min.png">
+                        <strong class="black">CashTan</strong>
+                    </a>
+                </div>
+                <div class="center-menu">
+                    <a class="point">*</a>
+                    <a class="general hat" href="#">ГЛАВНАЯ</a>
+                    <a class="point">*</a>
+                    <a class="catalog-shop hat" href="#">КАТАЛОГ МАГАЗИНОВ</a>
+                    <a class="point">*</a>
+                </div>
+                <div class="right-menu">
+                    <a class="entrance hat" href="#">ВХОД</a>
+                    <a class="point">*</a>
+                    <a class="checkin hat" href="#">РЕГИСТРАЦИЯ</a>
+
+                </div>
+            </div>
+
+        </div>
+        <!--личный кабинет-->
+        <div class="section">
+            <div class="title office">ЛИЧНЫЙ КАБИНЕТ</div>
+            <div class="privat">
+                <div class="form">
+                    <div class="my-profile">
+                        <div class="link-office">Мой профиль</div>
+                        <div class=" data data-profile">
+
+
+                            <div class="avatar">
+                                <div class="download">
+                                    <a href="javascript:showFrm()">Загрузить фото</a>
```

Comment:
Is using js here nessesary? Maybe implement it throug label with property 'for'

Reply:
Js need to remain. otherwise, there will be a button, and this is impractical